### ISSUE TITLE
New Installation issues 

### ISSUE THREAD

**Issue by tbaror** [MysticRyuujin/guac-install#181]
Hello,
I am trying to install on ubunto 20.4 lts, right after :
`Running Make on Guacamole-Server. This might take a few minutes...
Failed. See /tmp/guacamole_1.3.0_build.log
`
get error , looking in the log i see following error shown below , any idea?
Please advice
Thanks
`rdp.c: In function ‘guac_rdp_handle_connection’:
rdp.c:440:5: error: ‘VerifyCertificate’ is deprecated [-Werror=deprecated-declarations]
  440 |     rdp_inst->VerifyCertificate = rdp_freerdp_verify_certificate;
      |     ^~~~~~~~
In file included from /usr/include/freerdp2/freerdp/api.h:23,
                 from /usr/include/freerdp2/freerdp/freerdp.h:42,
                 from beep.h:23,
                 from rdp.c:21:
/usr/include/freerdp2/freerdp/freerdp.h:368:47: note: declared here
  368 |   WINPR_DEPRECATED(ALIGN64 pVerifyCertificate VerifyCertificate); /**< (offset 51)
      |                                               ^~~~~~~~~~~~~~~~~
/usr/include/winpr2/winpr/winpr.h:53:31: note: in definition of macro ‘WINPR_DEPRECATED’
   53 | #define WINPR_DEPRECATED(obj) obj __attribute__((deprecated))
      |                               ^~~
cc1: all warnings being treated as errors
make[4]: *** [Makefile:1329: libguac_client_rdp_la-rdp.lo] Error 1
make[4]: Leaving directory '/root/guacamole-server-1.3.0/src/protocols/rdp'
make[3]: *** [Makefile:1446: all-recursive] Error 1
make[3]: Leaving directory '/root/guacamole-server-1.3.0/src/protocols/rdp'
make[2]: *** [Makefile:748: all] Error 2
make[2]: Leaving directory '/root/guacamole-server-1.3.0/src/protocols/rdp'
make[1]: *** [Makefile:533: all-recursive] Error 1
make[1]: Leaving directory '/root/guacamole-server-1.3.0'
make: *** [Makefile:455: all] Error 2`

**Comment by wadegib** [comment]
I am getting the same errors as OP when building from a new Ubuntu 20.04.2 LTS server. I added the remmina repo and installed freerdp2-dev and freerdp2-x11 prior to installation as noted in the README but still get the same error. I'm not sure how to fix or bypass this error. TIA

**Comment by MysticRyuujin** [comment]
> I added the remmina repo and installed freerdp2-dev and freerdp2-x11 prior to installation as noted in the README

Did you try to install WITHOUT adding the custom repos? Those instructions are for people whom it ISN'T working for...

**Comment by wadegib** [comment]
> > I added the remmina repo and installed freerdp2-dev and freerdp2-x11 prior to installation as noted in the README
> 
> Did you try to install WITHOUT adding the custom repos? Those instructions are for people whom it ISN'T working for...

Yes, I was following a YT tutorial that had that step unnecessarily. I removed the custom repo and the script flew through. TY

**Comment by CorvenDallas-10** [comment]
Hi there Chase, I've been trying your script on Ubuntu 22.04 with Guacamole 1.4.0
I only had to make a modification in the download section because it wasn't downloading anymore. However, I do get the same error message as mentioned by @tbaror 
I realised as well that I made the error of adding the remmina repo. So I started over without the repo, and I got to the exact same spot, with the exact same error. Any pointers?
![image](https://user-images.githubusercontent.com/104764412/166220687-6477eb07-381c-40b5-b26f-d2eaf719c16d.png)
